modules/boost.units/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "homepage": "http://boost.org/libs/units",
+    "maintainers": [
+        {
+            "email": "daisuke.nishimatsu1021@gmail.com",
+            "github": "wep21",
+            "name": "Daisuke Nishimatsu"
+        },
+        {
+            "email": "julian.amann@tum.de",
+            "github": "Vertexwahn",
+            "name": "Julian Amann"
+        }
```

Comment:
@wep21 @Vertexwahn I added you as the maintainers since you're the maintainers for all the other boost libs. But please let me know if that is not appropriate here.

Reply:
Fine for me - feel free to add yourself!